Description: insert missing tag Keywords and some other changes
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Forwarded: not-needed
Last-Update: 2014-10-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/src/xsane.desktop
===================================================================
--- trunk.orig/src/xsane.desktop
+++ trunk/src/xsane.desktop
@@ -1,9 +1,19 @@
 [Desktop Entry]
-Encoding=UTF-8
-Name=XSane - Scanning
-Comment=Acquire images from a scanner
+Name=XSane Image scanning program
+Name[ru]=Программа для сканирования изображений
+Name[ca]=Programa d'escaneig d'imatges XSane
+Name[es]=Programa de escaneo de imágenes XSane
+Name[fr]=Programme d'acquisition d'images XSane
+Comment=A program to work with scanner. Can be used as a scanning, copier, OCR, fax tools.
+Comment[ru]=Программа для работы со сканером. Может быть использована в качестве копировального аппарата, факса, и др.
+Comment[ca]=Un programa per a treballar amb escànners. Es pot utilitzar com una eina d'escanejat, copiat, OCR i fax.
+Comment[es]=Un programa para trabajar con escáners. Se puede utilizar como una herramienta para escanear, copiar, OCR y fax.
+Comment[fr]=Un programme d'acquisition d'images pour votre scanner. Peut également photocopier, faxer ou faire de la reconnaissance de caractères.
 Exec=xsane
 Icon=xsane
 Terminal=false
 Type=Application
-Categories=Application;Graphics
+Categories=Application;Graphics;GTK;RasterGraphics;Scanning;OCR;2DGraphics;
+Keywords=scannerfrontend
+StartupNotify=true
+
